app/assets/scripts/components/csv-btn.js

```diff
@@ -131,16 +41,15 @@ var CSVBtn = React.createClass({
       csv = csv.concat(serialize.serialize(ministryActiveProjects));
     }
 
-    if (this.props.servedByProject) {
-      let servedByProject = serialize.chartData(this.props.servedByProject);
-      csv += '\n\nNumber Served Per Project\n\n';
-      csv = csv.concat(serialize.serialize(servedByProject));
-    }
+    this.props.chartData.forEach(d => {
```

Comment:
This should be `get(this.props, 'chartData', []).forEach(...` otherwise it'll throw errors on pages without charts.